kernel/update_kernel.sh

```diff
@@ -15,10 +15,10 @@ VERSION=$clear_vm_kernel_version
 RELEASE=$(cat release)
 BUILD_DISTROS=(Fedora_26 xUbuntu_16.04)
 
-KR_CONFIG_FILENAME=kernel-config-4.9.x
 KR_REL=https://www.kernel.org/releases.json
 KR_SHA=https://cdn.kernel.org/pub/linux/kernel/v4.x/sha256sums.asc
-KR_LTS=4.9
+KR_LTS=$(echo $VERSION | cut -d "." -f 1,2)
+KR_CONFIG_FILENAME=kernel-config-"${KR_LTS}".x
 
 GENERATED_FILES=(linux-container.dsc linux-container.spec _service config)
 STATIC_FILES=(debian.dirs debian.rules debian.compat debian.control debian.copyright patches-4.9.x/*.patch)
```

Comment:
I know this isn't part of the PR, but seeing the `KR_LTS` change above make me wonder if we should change this to avoid having to hard-code the `4.9` here. Something like:

```
patch_dir_pattern="patches-${KR_LTS}*"

patches=$(eval find "$patch_dir_pattern" -type f -name "*.patch")
```